product/product.css

```diff
@@ -0,0 +1,232 @@
+.container-nav {
+  position: sticky;
+  top: 0;
+  padding: 0;
+  grid-area: nav;
+}
+.logo {
+  height: 4.5rem;
+  width: 4.5rem;
+}
+
+.nav header {
+  padding: var(--gap-medium);
+  background-color: var(--background-color);
+}
+.nav .brand-link:hover {
+  background-color: var(--background-color);
+  opacity: 163%;
+}
+
+.name {
+  color: var(--primary-color);
+  margin: 0;
+}
+.nav .name {
+  margin-left: 0;
+}
+.search button {
+  background-color: var(--primary-color);
+}
+.menu li {
+  padding: var(--gap-small);
+}
+.menu a {
+  padding: var(--gap-small);
+  background-color: var(--primary-color);
+}
+
+/* container main */
+
+.container-main {
+  display: grid;
+  grid-template-columns: 15rem 1fr;
+}
+
+/* aside */
+aside {
+  position: sticky;
+  top: 6.5rem;
+
+  height: calc(100vh - (6.5rem));
+
+  background-color: var(--secondary-color);
+}
+
+.filters {
+  justify-content: start;
+  align-items: stretch;
+  gap: 2rem;
+
+  padding: var(--gap-small);
+}
+
+.filter {
+  justify-content: space-between;
+}
+
+.filter a {
+  color: var(--primary-color);
+}
+.range {
+  justify-content: space-between;
+}
+.slider input {
+  width: 100%;
+}
+/* slider */
+
+.container-range {
+  padding: var(--gap-small);
+  border-radius: 10px;
+}
+.slider input {
+  -webkit-appearance: none;
+  height: 3px;
+  width: 100%;
+  background-color: var(--background-color);
+  border-radius: 5px;
+  outline: none;
+  border: none;
+}
+.slider input::-webkit-slider-thumb {
+  -webkit-appearance: none;
+  height: 1rem;
+  width: 1rem;
+  background: var(--primary-color);
+  border-radius: 50%;
+  border: 1px solid var(--primary-color);
+  cursor: pointer;
+}
+/* checkbox */
+
+.checkboxes {
+  align-items: normal;
+}
+
+.checkbox {
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+  position: relative;
+}
+
+.checkbox > span {
+  padding: var(--gap-smallest);
+}
+
+.checkbox > input {
```

Comment:
Use a class instead of using child selector.

Reply:
yes ive fixed that